README.md

```diff
@@ -9,7 +9,7 @@ https://dfir-orc.github.io
 ## Build
 
 ### Requirements
-- Visual Studio >=2017 with this [configuration](.vsconfig) or alternatively use [vstools](docs/vstools/vstools.md)
+- Visual Studio >=2017 with this [configuration](.vsconfig) and "Desktop Developement C++" or alternatively use [vstools](docs/vstools/vstools.md)
```

Comment:
Why? Are you sure this requirement is not already selected via the .vsconfig ?

Reply:
With a fresh VS2017 install the import of .vsconfig file generate some warning and the "Desktop Developement C++" component are not checked.